webapp-django/crashstats/topcrashers/forms.py

```diff
@@ -0,0 +1,14 @@
+from django import forms
+
+from crashstats.supersearch.form_fields import MultipleValueField
```

Comment:
Perhaps we should refactor this and move this class into `crashstats/base/` since it's share between at least two apps (`supersearch` and `topcrashers`)